cognite/model_hosting/_cognite_model_hosting_common/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.2"
+__version__ = "0.1.3"
```

Comment:
You can go alpha first to test it in Nostromo before all clients start using it.